docs/releasing.md

```diff
@@ -66,11 +66,16 @@ git push -f origin stable
 
 If this is GA:
 
-* [ ] Update the [Homebrew tap](https://github.com/argoproj/homebrew-tap).
-* [ ] Check Homebrew was successfully updated:
+* [ ] Update the Homebrew formula.
+
+```bash
+brew bump-formula-pr argo --version $VERSION
```

Comment:
what does this to please?

Reply:
This downloads $VERSION of Argo based on the current formula, determines the sha256 of the download, then outputs the diff to the user. When he confirms this a branch in the users fork of the homebrew-core repository is created and a pull request to the original homebrew-core is opened. Also a browser window is opened showing the pull request. This makes it very convenient to update formulas.
For the Argo CD project I even supplied a GitHub Action that does this automatically when you make a GitHub release of Argo CD. We could do the same for the Argo CLI.

Reply:
I also have a personal repository that automatically updates a bunch of Homebrew formulas in homebrew-core running every 6 hours. If you don't want to automate this for now I will simply include the Argo CLI in that list.

Reply:
@alexec Are you missing anything else?